migration/transitioning.rst

```diff
@@ -0,0 +1,92 @@
+.. _transitioning_mt:
+
+Migrating an existing single-node PostgreSQL database to Citus sometimes requires adjusting the schema and queries for optimal performance. Citus extends PostgreSQL with distributed functionality, but it is not a drop-in replacement that scales out all workloads. A performant Citus cluster involves thinking about the data model, tooling, and choice of SQL features used.
```

Comment:
As noted, we aren't making a specific recommendation here around migration. So we can just say something like "Migrating an existing solution to Citus" or "Migrating an existing relational store to Citus".